x/capability/abci.go

```diff
@@ -0,0 +1,21 @@
+package capability
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/capability/keeper"
+	"github.com/cosmos/cosmos-sdk/x/capability/types"
+)
+
+// BeginBlocker will call InitMemStore to initialize the memory stores in the case
+// that this is the first time the node is executing a block since restarting (wiping memory).
+// In this case, the BeginBlocker method will reinitialize the memory stores locally, so that subsequent
+// capability transactions will pass.
+// Otherwise BeginBlocker performs a no-op.
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
```

Comment:
Do we need this function? We can move it's content to `AppModule.BeginBlocker`. In tests we don't call this function directly. And if we want to test keeper only, the we can still do `k.InitMemStore`.

Reply:
Yes I can remove